### ISSUE TITLE
Verification Info: As an Industry Reporter, I want to fill out the LFO verification form

### ISSUE THREAD

**Issue by shon-button** [bcgov/cas-registration#2470]
#### Description:

All Regulated Operations must fill out this page. 
Reporting Operations who have emissions above 25,000 T CO2e will also need to do this step (reporting operations under 25,000T will not). 

[LFO Wireframe](https://www.figma.com/design/LsDmLDCdnJqI8UahyqvOD8/BCIERS-Designs?node-id=17438-4851&t=kkLCs25GtG7LwPro-4)

#### Acceptance Criteria:

Given I am a REPORTING Operation who has attributable emissions greater than or equal to 25,000 T CO2e (not including CO2 from biomass listed in item 1 of Schedule C)
Then I need to fill out the verification LFO form page. 

#### Development Checklist:

- [ ] Update front-end schemas to manage array of sites visited; array of types of site visited `bciers/apps/reporting/src/data/jsonSchema/signOff/verification/verification.ts`
- [ ] Update model to accept array of sites visited in file `bc_obps/reporting/models/report_verification.py`
- [ ] Update schema to accept array of sites visited in file `bc_obps/reporting/schema/report_verification.py`
- [ ] Update service to save array of sites visited; array of types of site visited `bc_obps/reporting/service/report_verification_service.py`
- [ ] ???
- [ ]  Meets the DOD

**Definition of Ready** (Note: If any of these points are not applicable, mark N/A)

- [ ] User story is included
- [ ] User role and type are identified
- [ ] Acceptance criteria are included
- [ ] Wireframes are included (if required)
- [ ] Design / Solution is accepted by Product Owner
- [ ] Dependencies are identified (technical, business, regulatory/policy)
- [ ] Story has been estimated (under 13 pts)

**Definition of Done** (Note: If any of these points are not applicable, mark N/A)

- [ ] Acceptance criteria are tested by the CI pipeline
- [ ] UI meets accessibility requirements
- [ ] Configuration changes are documented, documentation and designs are updated
- [ ] Passes code peer-review
- [ ] Passes QA of Acceptance Criteria with verification in Dev and Test
- [ ] Ticket is ready to be merged to main branch
- [ ] Can be demoed in Sprint Review
- [ ] Bugs or future work cards are identified and created
- [ ] Reviewed and approved by Product Owner

#### Notes:

- Business Area creating a template for this so we're blocked on this page. 
#200 blocked by this

**Comment by shon-button** [comment]
@patriciarussellCAS 

This ticket is specific to LFO; so, a split form the existing ticket.
